tests/usbus_hid/main.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (C) 2020 Nils Ollrogge
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     unittests
```

Comment:
this field should be update.

Reply:
change it to tests ? I that what you meant?

Reply:
Yes